lib/datadog/ci/ext/environment.rb

```diff
@@ -1,4 +1,5 @@
 # typed: true
+
 # frozen_string_literal: true
```

Comment:
These should be grouped together I suppose. I can't recall exactly what but I seem to recall that there was some ordering importance for these.

Reply:
You, dear sir, drive a hard bargain! Fixed in 9964260e1ee2af834b6b30a8c6f3c5f58d8d8d5b . I actually think it works this way as well, but it was faster to change the handful of files than to chase down the answer :)

Reply:
Actually had to split them again because rubocop was complaining, but I left `frozen_string_literal` at the top.

Reply:
Ah yes, Rubocop is uselessly annoying on that one!